master/getting-started/kubernetes/installation/index.md

```diff
@@ -42,3 +45,32 @@ You can find some of them here, organized by cloud provider.
 - [Amazon Web Services](aws)
 - [Google Compute Engine](gce)
 - [Microsoft Azure](azure)
+
+## <a id="ipvs-mode"></a>kube-proxy IPVS mode
```

Comment:
Actually, you don't need to manually add any HTML to achieve the anchor links. These are created automatically by adding hyphens between each word in the heading and all lowercase. There are some slightly interesting aspects with special characters but overall, that's how it works. So this heading could be referenced as `#kube-proxy-ipvs-mode`